tests/Unit/Services/Contact/Contact/CreateContactTest.php

```diff
@@ -48,6 +48,39 @@ public function test_it_stores_a_contact()
         );
     }
 
+    public function test_it_stores_a_contact_without_gender()
```

Comment:
Here perhaps we could add a test asserting that the contact effectively has a gender_id set to null.

Reply:
done